composer.json

```diff
@@ -35,7 +35,7 @@
         "psr/http-server-handler": "^1.0",
         "psr/http-server-middleware": "^1.0",
         "psr/log": "^3.0",
-        "psr/simple-cache": "^1.0.0"
+        "psr/simple-cache": "^3.0"
```

Comment:
We can probably support 2 || 3 if 2 is the parameter changes.

Reply:
Done